utils/chat_utils.py

```
@@ -15,124 +20,68 @@ def fetch_user_personality(user_id):
+        f"시간: {course_dict.start_time} ~ {course_dict.end_time}\n"
+        f"금액: {course_dict.price}원\n"
+        f"카테고리: {course_dict.main_category} / {course_dict.sub_category}\n"
+        f"인원: {course_dict.headcount}\n"
+        f"태그: {course_dict.tags}\n"
     )
-    return message, course_dict.get("프로그램명", None)
+    return message, course_dict.name
 
 
-def recommend_random_program(user_id):
+def recommend_random_program(user_id: int, db:Session=Depends(get_db)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)